src/frontend/youtube_clone/src/app/app.module.ts

```diff
@@ -1,16 +1,36 @@
 import { NgModule } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
-
+import { MatSlideToggleModule } from '@angular/material/slide-toggle';
 import { AppRoutingModule } from './app-routing.module';
+
 import { AppComponent } from './app.component';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { MainPageComponent } from './main-page/main-page.component';
+import { VideoPageComponent } from './video-page/video-page.component';
+import { ProfilePageComponent } from './profile-page/profile-page.component';
+import { RouterModule } from '@angular/router';
+import {MatSidenavModule} from '@angular/material/sidenav';
 
 @NgModule({
   declarations: [
-    AppComponent
+    AppComponent,
+    MainPageComponent,
+    VideoPageComponent,
+    ProfilePageComponent,
   ],
   imports: [
     BrowserModule,
-    AppRoutingModule
+    AppRoutingModule,
+    BrowserAnimationsModule,
+    MatSlideToggleModule,
+    MatSidenavModule,
+    RouterModule.forRoot([      
```

Comment:
It's a bad practice to have all routes inline in `AppModule`. Create separate `AppRoutingModule`

Reply:
So you fixed it in a different PR?